src/integration-tests/integration-test.py

```diff
@@ -97,6 +98,7 @@ class Integration(BaseModel):
         target_exe="fuzz.exe",
         inputs="seeds",
         wait_for_files={ContainerType.unique_reports: 1},
+        pool="linux",
```

Comment:
The pool should not be hardcoded here. the test should be independent of the pool as long as it has the right OS. This will allow us to run all linux tests on mariner

Reply:
We can still run all linux tests on mariner, I just need to update the integration tests lists to add those tests. How are you imagining that we select which OS a test should run on?

Reply:
> We can still run all linux tests on mariner, I just need to update the integration tests lists to add those tests

looks like we will end up diplicating the linux definitions for the mariner.  Ideally we shoudl use the exact same definition

> How are you imagining that we select which OS a test should run on?

I was thinking of having an extra paramter in the target definition which would be the list of OS flabors where this test can run. Something like 

```python
"linux-trivial-crash-afl": Integration(
        template=TemplateType.afl,
        os=OS.linux,
        target_exe="fuzz.exe",
        inputs="seeds",
        wait_for_files={ContainerType.unique_reports: 1},
        os_flavors=["mariner", "ubuntu"],
)
```
and use this information to select the pool where it can run. But ultimately this might be a bigger change than i anticipated. So, take this as just a non-blocking suggestion.